Makefile

```diff
@@ -78,3 +78,12 @@ curl_local curl_prod:
 		"N":["C7", "H6", "H7", "H9", "CJ", "SA", "S8", "SQ", "D5", "S5", "HK", "C8", "HA"],    \
 		"E":["H2", "H5", "CQ", "D9", "H4", "ST", "HQ", "SJ", "HJ", "DQ", "H3", "C9", "CK"] }}' \
 	${CURL_URL}
+
+logfile := logs/src.api.$(shell date +%FT%T).log
+pidfile := logs/server.PID
+
+start_local_server: libdds-build stop_local_server
+	python3 -m src.api > ${logfile} 2>&1 & echo $$! > ${pidfile}
+
+stop_local_server:
+	[ ! -e ${pidfile} ] || (kill $$(cat ${pidfile}) ; rm ${pidfile})
```

Comment:
There seem to be no new line at the end of file. Git likes to warn about some whitespace errors.

Reply:
Fixed.

I've now instructed PyCharm to take care of this automatically.